bears/documentation/DocumentationStyleBear.py

```diff
@@ -134,6 +134,11 @@ def run(self, filename, file, language: str,
                 new_metadata, doc_comment.docstyle_definition,
                 doc_comment.marker, doc_comment.indent, doc_comment.position)
 
+            if file[doc_comment.range.start.line - 2] != '\n':
```

Comment:
Ahh I didnt get you. Yup we only want this. To follow PEP-257 standard, we just check if there's a blank line before and after the docstring. If its there then good if not we apply a fix. Also I would like to have your idea about padding applied in DocumentationAPI. Which will give diversity to other developers :tada: